### PR TITLE
Chaos Labs Risk Parameter Updates - Aave V3 Optimism

### PR DESCRIPTION
**Pull request by Natanela**
Discussion: [https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466/1](https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466/1)

### CODE REVIEW

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import {AaveV3PayloadOptimism, IEngine, EngineFlags} from 'aave-helpers/v3-config-engine/AaveV3PayloadOptimism.sol';
+
+/**
+ * @title Chaos Labs Risk Parameter Updates - Aave V3 Optimism - 2023.08.13
+ * @author Chaos Labs
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x7568e17d2d7078686255a8fadf563e1f072abae0b79188a5b5b76852a6ebd63f
+ * - Discussion: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466/1
+ */
+contract AaveV3_Opt_RiskParamsUpdate_20232408 is AaveV3PayloadOptimism {
+  function collateralsUpdates() public view virtual override returns (IEngine.CollateralUpdate[] memory) {
```
Comment (yonikesel1) [inline src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.sol]:
```suggestion
  function collateralsUpdates() public pure override returns (IEngine.CollateralUpdate[] memory) {
```

Comment (yonikesel1) [reply src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.sol]: Should be pure function

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/RiskParamsUpdate.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Based on our analysis, we have determined that there is adequate on-chain liquidity to support any significant liquidations with the updated parameters.
+
+### Positions Analysis
+
+We have not identified any outsized positions that are actively affecting our recommendations.
+
+### Recommendations
+
+| Asset | Parameter | Current | Recommended | Change |
```
Comment (yonikesel1) [inline src/AaveV3_Opt_RiskParamsUpdate_20232408/RiskParamsUpdate.md]: please run formatter on all files

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/RiskParamsUpdate.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Chaos Labs Risk Parameter Updates - Aave V3 Optimism - 2023.08.13
+author: Chaos Labs
+discussions: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466/1
```
Comment (yonikesel1) [inline src/AaveV3_Opt_RiskParamsUpdate_20232408/RiskParamsUpdate.md]:
```suggestion
discussions: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466
```

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/RiskParamsUpdate.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: Chaos Labs Risk Parameter Updates - Aave V3 Optimism - 2023.08.13
```
Comment (yonikesel1) [inline src/AaveV3_Opt_RiskParamsUpdate_20232408/RiskParamsUpdate.md]:
```suggestion
title: Chaos Labs Risk Parameter Updates - Aave V3 Optimism
```

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.t.sol [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    e2eTestAsset(
+    AaveV3Optimism.POOL,
+    _findReserveConfig(allConfigsAfter, AaveV3OptimismAssets.USDC_UNDERLYING),
+    _findReserveConfig(allConfigsAfter, AaveV3OptimismAssets.wstETH_UNDERLYING)
+    );
+    e2eTestAsset(
+    AaveV3Optimism.POOL,
+    _findReserveConfig(allConfigsAfter, AaveV3OptimismAssets.USDC_UNDERLYING),
+    _findReserveConfig(allConfigsAfter, AaveV3OptimismAssets.WBTC_UNDERLYING)
+    );
```
Comment (yonikesel1) [inline src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.t.sol]:
```suggestion
    e2eTest(AaveV3Optimism.POOL);
```

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.sol [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+pragma solidity ^0.8.0;
+
+import {AaveV3OptimismAssets} from 'aave-address-book/AaveV3Optimism.sol';
+import {AaveV3PayloadOptimism, IEngine, EngineFlags} from 'aave-helpers/v3-config-engine/AaveV3PayloadOptimism.sol';
+
+/**
+ * @title Chaos Labs Risk Parameter Updates - Aave V3 Optimism - 2023.08.13
+ * @author Chaos Labs
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x7568e17d2d7078686255a8fadf563e1f072abae0b79188a5b5b76852a6ebd63f
+ * - Discussion: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466/1
```
Comment (yonikesel1) [inline src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.sol]:
```suggestion
 * - Discussion: https://governance.aave.com/t/arfc-chaos-labs-risk-parameter-updates-aave-v3-optimism-2023-08-13/14466
```

File: src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.t.sol (hunk 5 above)
Comment (yonikesel) [reply src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.t.sol]: e2eTest should work now, can you change to it and run the test again?

Comment (Natanela) [reply src/AaveV3_Opt_RiskParamsUpdate_20232408/AaveV3_Opt_RiskParamsUpdate_20232408.t.sol]: done. it worked

**Review by yonikesel — APPROVED** [review]

**Pull request closed without merging**